netbox/extras/api/serializers.py

```diff
@@ -58,22 +59,38 @@
 
 
 #
-# Webhooks
+# Event Rules
 #
 
-class WebhookSerializer(NetBoxModelSerializer):
-    url = serializers.HyperlinkedIdentityField(view_name='extras-api:webhook-detail')
+class EventRuleSerializer(NetBoxModelSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name='extras-api:eventrule-detail')
     content_types = ContentTypeField(
         queryset=ContentType.objects.filter(FeatureQuery('webhooks').get_query()),
```

Comment:
This will need to be updated for #14153 and to reference "event rules" as a feature.